### ISSUE TITLE
[Bug]: The "Language of UI" will Toggle to Chinese but not to English

### ISSUE THREAD

**Issue by DavidDragonsage** [metercai/SimpleSDXL#80]
### Checklist

- [X] The issue has not been resolved by following the [troubleshooting guide](https://github.com/lllyasviel/Fooocus/blob/main/troubleshoot.md)
- [X] The issue exists on a clean installation of Fooocus
- [X] The issue exists in the current version of Fooocus
- [X] The issue has not been reported before recently
- [ ] The issue has been reported before but has not been fixed yet

### What happened?

Under the Enhanced tab, the Language of UI selector only works one way. If you select the Chinese language it changes the UI to Chinese. If you try to switch back to English the UI remains in Chinese.

### Steps to reproduce the problem

1) Go to the "Language of UI" section under the Enhanced tab.
2) If the UI is currently in Chinese, the "En" selector does nothing.
3) If the UI is currently in English, the Chinese selector works properly and the UI changes to Chinese.

### What should have happened?

The Language of UI selectors should work properly, allowing the user to switch freely between either of the two supported languages.

### What browsers do you use to access Fooocus?

Firefox

### Where are you running Fooocus?

Locally

### What operating system are you using?

Windows 10

### Console logs

```Shell
Not applicable: the console does not record the activity of the Language of UI selectors.
```

### Additional information

While adding the "--language en" language switch to run.bat does work, that is not convenient when doing a new installation using install_and_run_安装模型再启动.bat - and I seem to have been doing a lot of new installations lately!

**Comment by DavidDragonsage** [comment]
The images show the UI starting in English mode, then toggled to Chinese successfully, then the toggle back to English is unsuccessful.

![SimpleSDXL in English mode](https://github.com/user-attachments/assets/21e

![SimpleSDXL in Chinese mode](https://github.com/user-attachments/assets/630cdff1-324f-48ae-96fc-7c5057123493)

![SimpleSDXL will not Toggle Back](https://github.com/user-attachments/assets/27818e88-a664-4da4-a42a-297ba2e99cda)

**Comment by metercai** [comment]
Known reserved bug, waiting to be fixed. But it's not an urgent matter.

**Comment by DavidDragonsage** [comment]
Agreed, there are higher priorities for sure. That is good to know the status, thank you.

**Comment by nyukers** [comment]
Use --language en

**Comment by DavidDragonsage** [comment]
@nyukers  Yes, but that is not quite the point - please check the "Additional Information" section in the issue list. Happy holidays, nyukers! ❤️

**Comment by metercai** [comment]
你们自己去解决吧，SimpleSDXL是中文优先的
You can solve it yourselves. SimpleSDXL prioritizes Chinese
